src/database/accounts.js

```diff
@@ -51,29 +51,16 @@ module.exports = {
     },
 
     /**
-     * Search for a user by name
+     * Search for a user by any field
      */
-    search: function (name, fields, callback) {
-        /* This bit allows it to accept varargs
```

Comment:
"Varags" is poorly worded, I think my reasoning here was to make the `fields` parameter optional and default to just selecting the name.  But I don't know if this is actually used.